src/core/client/stream/tabs/Comments/Comment/InReplyTo.tsx

```diff
@@ -2,21 +2,53 @@ import { Localized } from "@fluent/react/compat";
 import cn from "classnames";
 import React, { FunctionComponent } from "react";
 
+import { useCoralContext } from "coral-framework/lib/bootstrap";
 import CLASSES from "coral-stream/classes";
+import computeCommentElementID from "coral-stream/tabs/Comments/Comment/computeCommentElementID";
 import { Flex, Icon } from "coral-ui/components/v2";
 
+import { CommentContainer_comment as CommentData } from "coral-stream/__generated__/CommentContainer_comment.graphql";
+
 import styles from "./InReplyTo.css";
 
 interface Props {
-  username: string;
+  parent: CommentData["parent"];
+  isLink: boolean;
 }
 
-const InReplyTo: FunctionComponent<Props> = ({ username }) => {
-  const Username = () => (
-    <span className={cn(styles.username, CLASSES.comment.inReplyTo.username)}>
-      {username}
-    </span>
-  );
+const InReplyTo: FunctionComponent<Props> = ({ parent, isLink }) => {
+  const { pym } = useCoralContext();
+
+  const navigateToParent = (id: string) => {
+    /* eslint-disable */
+    console.log('navigating to parent');
+    const elemId = computeCommentElementID(id);
+    const elem = document.getElementById(elemId);
+    console.log(elem);
+    if (elem) {
+      void pym?.scrollParentToChildEl(elemId);
+      elem.focus();
+      (elem as any).style["border"] = "5px solid green";
```

Comment:
Thank you for using pym! This is one of the main things I was looking for! 💯

Reply:
Make sure to run this border thing by design to check this is what they want. Super helpful for testing though!